po/sv_SE.po

```diff
@@ -0,0 +1,972 @@
+msgid "cyan"
+msgstr "turkos"
+
+#: rc.c:284
+msgid "white"
+msgstr "vitt"
+
+#: rc.c:285
+msgid "terminal"
+msgstr "terminal"
+
+#: rc.c:304
+msgid "none"
+msgstr "ingen"
+
+#: rc.c:305
+msgid "current URL"
+msgstr "aktuell URL"
+
+#: rc.c:306
+msgid "link URL"
+msgstr "länk-URL"
+
+#: rc.c:311
+msgid "simple"
+msgstr "enkel"
+
+#: rc.c:312
+msgid "use tag"
+msgstr "använd tagg"
+
+#: rc.c:313
+msgid "fontify"
+msgstr "fontify"
+
+#: rc.c:319
+msgid "A:relative to screen height"
+msgstr "A:relativ till skärmhöjd"
+
+#: rc.c:320
+msgid "B:fixed speed"
+msgstr "B:fix hastighet"
+
+#: rc.c:327
+msgid "unspecified"
+msgstr "ospecificerat"
+
+#: rc.c:328
+msgid "inet inet6"
+msgstr "inet inet6"
+
+#: rc.c:329
+msgid "inet6 inet"
+msgstr "inet6 inet"
+
+#: rc.c:330
+msgid "inet only"
+msgstr "enbart inet"
+
+#: rc.c:331
+msgid "inet6 only"
+msgstr "enbart inet6"
+
+#: rc.c:338
+msgid "discard"
+msgstr "förkasta"
+
+#: rc.c:340
+msgid "accept"
+msgstr "acceptera"
+
+#: rc.c:342
+msgid "ask"
+msgstr "fråga"
+
+#: rc.c:349
+msgid "use internal mailer instead"
+msgstr "använd internt epostprogram istället"
+
+#: rc.c:351
+msgid "ignore options and use only the address"
+msgstr "ignorera alternativ och använd endast adressen"
+
+#: rc.c:352
+msgid "use full mailto URL"
+msgstr "använd fullständig mailto-URL"
+
+#: rc.c:361
+msgid "OFF"
+msgstr "AV"
+
+#: rc.c:362
+msgid "Only ISO 2022"
+msgstr "Endast ISO 2022"
+
+#: rc.c:363
+msgid "ON"
+msgstr "PÅ"
+
+#: rc.c:369
+msgid "ASCII"
+msgstr "ASCII"
+
+#: rc.c:370
+msgid "charset specific"
+msgstr "teckenuppsättningsspecifik"
+
+#: rc.c:371
+msgid "DEC special graphics"
+msgstr "DEC-specialgrafik"
+
+#: rc.c:377
+msgid "external command"
+msgstr "externt kommando"
+
+#: rc.c:378
+msgid "OSC 5379 (mlterm)"
+msgstr "OSC 5379 (mlterm)"
+
+#: rc.c:379
+msgid "sixel (img2sixel)"
+msgstr "sixel (img2sixel)"
+
+#: rc.c:380
+msgid "OSC 1337 (iTerm2)"
+msgstr "OSC 1337 (iTerm2)"
+
+#: rc.c:381
+msgid "kitty (ImageMagick)"
+msgstr "kitty (ImageMagick)"
+
+#: rc.c:779
+msgid "Display Settings"
+msgstr "Visningsinställningar"
+
+#: rc.c:781
+msgid "Color Settings"
+msgstr "Färginställningar"
+
+#: rc.c:783
+msgid "Miscellaneous Settings"
+msgstr "Diverse inställningar"
+
+#: rc.c:784
+msgid "Directory Settings"
+msgstr "Kataloginställningar"
+
+#: rc.c:785
+msgid "External Program Settings"
+msgstr "Inställningar för externa program"
+
+#: rc.c:786
+msgid "Network Settings"
+msgstr "Nätverksinställningar"
+
+#: rc.c:787
+msgid "Proxy Settings"
+msgstr "Proxyinställningar"
+
+#: rc.c:789
+msgid "SSL Settings"
+msgstr "SSL-inställningar"
+
+#: rc.c:792
+msgid "Cookie Settings"
+msgstr "Kakinställningar"
+
+#: rc.c:795
+msgid "Charset Settings"
+msgstr "Teckenuppsättningsinställningar"
+
+#. TRANSLATORS:
+#. * AcceptLang default: this is used in Accept-Language: HTTP request
+#. * header. For example, ja.po should translate it as
+#. * "ja;q=1.0, en;q=0.5" like that.
+#.
+#: rc.c:1277
+msgid "en;q=1.0"
+msgstr "en;q=1.0"
```

Comment:
Looking at the comment and examples at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language , I wonder if something like "sv-SE, sv;q=1.0, en;q=0.5" would be best here.

Reply:
I entirely missed the TRANSLATORS comment!